package.json

```diff
@@ -18,7 +19,7 @@
     "@matterlabs/hardhat-zksync-verify": "^0.2.1",
     "@matterlabs/zksync-contracts": "^0.6.1",
     "@nomiclabs/hardhat-etherscan": "^3.1.7",
-    "@openzeppelin/contracts": "^4.9.3",
+    "@openzeppelin/contracts": "^4.6.0",
```

Comment:
Why the downgrade?

Reply:
> Downgraded @openzeppelin/contracts to 4.6.0 to fix dependency versions missmatch (error when installing with npm)

You can try to install with `npm i` with version from main and it would fail if not using `--force` flag. The reason is that `@matterlabs/zksync-contracts` is using `@openzeppelin/contracts v4.6.0` and that was the problem

Reply:
ahh i see I see